src/main/resources/inspectionDescriptions/RustCStringPointer.html

```diff
@@ -0,0 +1,8 @@
+<html>
+<body>
+Checks unsafe usage of CString pointer.
+
+Corresponds to <a href="https://github.com/rust-lang/rfcs/pull/1642">add CStr::with_ptr and deprecate CStr::as_ptr</a>
+from Rust RFCs.
+</body>
+</html>
```

Comment:
Let's change this to

```
Detects unsafe usage of `CString`.

The code like `CString::new("hello").unwrap().as_ptr()` may lead to memory unsafety because the string is deallocated at the end of the expression. 
```